# Hungarian translation for glib-networking.
# Copyright (C) 2011, 2012, 2017, 2018, 2019, 2020, 2021. Free Software Foundation, Inc.
# This file is distributed under the same license as the glib-networking package.
#
# Gabor Kelemen <kelemeng at gnome dot hu>, 2011, 2012.
# Balázs Úr <ur.balazs at fsf dot hu>, 2012, 2017, 2018, 2019, 2020, 2021.
msgid ""
msgstr ""
"Project-Id-Version: glib-networking master\n"
"Report-Msgid-Bugs-To: https://gitlab.gnome.org/GNOME/glib-networking/issues\n"
"POT-Creation-Date: 2020-12-04 17:58+0000\n"
"PO-Revision-Date: 2021-01-19 21:13+0100\n"
"Last-Translator: Balázs Úr <ur.balazs at fsf dot hu>>\n"
"Language-Team: Hungarian <gnome-hu-list at gnome dot org>\n"
"Language: hu\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Lokalize 19.12.3\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: proxy/libproxy/glibproxyresolver.c:164
msgid "Proxy resolver internal error."
msgstr "Proxyfeloldó belső hiba."

#: tls/base/gtlsconnection-base.c:544 tls/base/gtlsinputstream.c:78
#: tls/base/gtlsinputstream.c:141 tls/base/gtlsoutputstream.c:78
#: tls/base/gtlsoutputstream.c:143
msgid "Connection is closed"
msgstr "A kapcsolat lezárva"

#. Cannot perform a blocking operation during a handshake on the
#. * same thread that triggered the handshake. The only way this can
#. * occur is if the application is doing something weird in its
#. * accept-certificate callback. Allowing a blocking op would stall
#. * the handshake (forever, if there's no timeout). Even a close
#. * op would deadlock here.
#.
#: tls/base/gtlsconnection-base.c:618
msgid "Cannot perform blocking operation during TLS handshake"
msgstr "A blokkoló művelet nem végezhető el a TLS-kézfogás alatt"

#: tls/base/gtlsconnection-base.c:683 tls/base/gtlsconnection-base.c:1225
msgid "Socket I/O timed out"
msgstr "A foglalat túllépte az I/O időkorlátot"

#: tls/base/gtlsconnection-base.c:851
msgid "Server required TLS certificate"
msgstr "A kiszolgáló TLS-tanúsítványt kért"

#: tls/base/gtlsconnection-base.c:1425
#, c-format
msgid "Handshake is not finished, no channel binding information yet"
msgstr "A kézfogás nem fejeződött be, még nincsenek csatornakötési információk"

#: tls/base/gtlsconnection-base.c:1484
msgid "Peer does not support safe renegotiation"
msgstr "A partner nem támogatja a biztonságos újraegyeztetést"

#: tls/base/gtlsconnection-base.c:1628 tls/gnutls/gtlsconnection-gnutls.c:428
#: tls/openssl/gtlsconnection-openssl.c:189
#: tls/openssl/gtlsconnection-openssl.c:648
#, c-format
msgid "Unacceptable TLS certificate"
msgstr "Elfogadhatatlan TLS-tanúsítvány"

#: tls/base/gtlsconnection-base.c:2093
#, c-format
msgid "Receive flags are not supported"
msgstr "A fogadási jelzők nem támogatottak"

#: tls/base/gtlsconnection-base.c:2245
#, c-format
msgid "Send flags are not supported"
msgstr "A küldési jelzők nem támogatottak"

#: tls/gnutls/gtlscertificate-gnutls.c:194
#: tls/openssl/gtlscertificate-openssl.c:170
#, c-format
msgid "Could not parse DER certificate: %s"
msgstr "A DER tanúsítvány nem dolgozható fel: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:215
#: tls/openssl/gtlscertificate-openssl.c:190
#, c-format
msgid "Could not parse PEM certificate: %s"
msgstr "A PEM tanúsítvány nem dolgozható fel: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:238
#: tls/openssl/gtlscertificate-openssl.c:209
#, c-format
msgid "Could not parse DER private key: %s"
msgstr "A DER személyes kulcs nem dolgozható fel: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:261
#: tls/openssl/gtlscertificate-openssl.c:228
#, c-format
msgid "Could not parse PEM private key: %s"
msgstr "A PEM személyes kulcs nem dolgozható fel: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:288
#, c-format
#| msgid "Could not parse PEM certificate: %s"
msgid "Could not import PKCS #11 certificate URI: %s"
msgstr "A PKCS #11 tanúsítvány URI nem importálható: %s"

#: tls/gnutls/gtlscertificate-gnutls.c:330
#: tls/openssl/gtlscertificate-openssl.c:263
msgid "No certificate data provided"
msgstr "Nincsenek megadva tanúsítványadatok"

#: tls/gnutls/gtlsconnection-gnutls.c:143
#: tls/gnutls/gtlsconnection-gnutls.c:161
#: tls/openssl/gtlsclientconnection-openssl.c:428
#: tls/openssl/gtlsserverconnection-openssl.c:480
#, c-format
msgid "Could not create TLS connection: %s"
msgstr "Nem sikerült létrehozni a TLS-kapcsolatot: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:372
#: tls/gnutls/gtlsconnection-gnutls.c:383
#: tls/gnutls/gtlsconnection-gnutls.c:397
#: tls/openssl/gtlsconnection-openssl.c:156
#, c-format
msgid "Peer failed to perform TLS handshake: %s"
msgstr "A partner nem tudta végrehajtani a TLS-kézfogást: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:405
msgid "TLS connection closed unexpectedly"
msgstr "A TLS-kapcsolat váratlanul befejeződött"

#: tls/gnutls/gtlsconnection-gnutls.c:420
#: tls/openssl/gtlsconnection-openssl.c:181
msgid "TLS connection peer did not send a certificate"
msgstr "A TLS kapcsolat partner nem küldött tanúsítványt"

#: tls/gnutls/gtlsconnection-gnutls.c:436
#, c-format
msgid "Peer sent fatal TLS alert: %s"
msgstr "A partner végzetes TLS riasztást küldött: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:446
msgid "Protocol version downgrade attack detected"
msgstr "Protokoll verzió visszaminősítéses támadás észlelve"

#: tls/gnutls/gtlsconnection-gnutls.c:455
#, c-format
msgid "Message is too large for DTLS connection; maximum is %u byte"
msgid_plural "Message is too large for DTLS connection; maximum is %u bytes"
msgstr[0] "Az üzenet túl nagy a DTLS kapcsolathoz; legfeljebb %u bájt lehet"
msgstr[1] "Az üzenet túl nagy a DTLS kapcsolathoz; legfeljebb %u bájt lehet"

#: tls/gnutls/gtlsconnection-gnutls.c:464
msgid "The operation timed out"
msgstr "A művelet túllépte az időkorlátot"

#: tls/gnutls/gtlsconnection-gnutls.c:796
#, c-format
msgid "Error performing TLS handshake: %s"
msgstr "Hiba a TLS-kézfogás végrehajtásakor: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:899
#: tls/openssl/gtlsconnection-openssl.c:427
#: tls/openssl/gtlsconnection-openssl.c:641
msgid "Error performing TLS handshake"
msgstr "Hiba a TLS-kézfogás végrehajtásakor"

#: tls/gnutls/gtlsconnection-gnutls.c:957
#, c-format
msgid "Channel binding type tls-unique is not implemented in the TLS library"
msgstr ""
"A tls-unique csatornakötési típus nincs megvalósítva a TLS "
"programkönyvtárban"

#: tls/gnutls/gtlsconnection-gnutls.c:961
#, c-format
msgid "Channel binding data for tls-unique is not yet available"
msgstr "A tls-unique csatornakötési adatok még nem érhetők el"

#: tls/gnutls/gtlsconnection-gnutls.c:988
#: tls/gnutls/gtlsconnection-gnutls.c:1000
#, c-format
msgid "X.509 certificate is not available on the connection"
msgstr "Az X.509 tanúsítvány nem érhető el a kapcsolaton"

#: tls/gnutls/gtlsconnection-gnutls.c:1013
#, c-format
msgid "X.509 certificate is not available or is of unknown format: %s"
msgstr "Az X.509 tanúsítvány nem érhető el vagy ismeretlen formátumú: %s"

#: tls/gnutls/gtlsconnection-gnutls.c:1024
#: tls/openssl/gtlsconnection-openssl.c:520
#, c-format
#| msgid "Unacceptable TLS certificate authority"
msgid "Unable to obtain certificate signature algorithm"
msgstr "A tanúsítvány aláírási algoritmusa nem kérhető le"

#: tls/gnutls/gtlsconnection-gnutls.c:1040
#: tls/openssl/gtlsconnection-openssl.c:540
#, c-format
msgid ""
"Current X.509 certificate uses unknown or unsupported signature algorithm"
msgstr ""
"A jelenlegi X.509 tanúsítvány ismeretlen vagy nem támogatott aláírási "
"algoritmust használ"

#: tls/gnutls/gtlsconnection-gnutls.c:1126
#: tls/openssl/gtlsconnection-openssl.c:620
#, c-format
msgid "Requested channel binding type is not implemented"
msgstr "A kért csatornakötési típus nincs megvalósítva"

#: tls/gnutls/gtlsconnection-gnutls.c:1147
#: tls/gnutls/gtlsconnection-gnutls.c:1207
#: tls/openssl/gtlsconnection-openssl.c:752
msgid "Error reading data from TLS socket"
msgstr "Hiba az adatok olvasásakor a TLS-foglalatból"

#. flags
#: tls/gnutls/gtlsconnection-gnutls.c:1229
#: tls/gnutls/gtlsconnection-gnutls.c:1292
#: tls/openssl/gtlsconnection-openssl.c:796
msgid "Error writing data to TLS socket"
msgstr "Hiba az adatok TLS-foglalatba írásakor"

#: tls/gnutls/gtlsconnection-gnutls.c:1262
#, c-format
msgid "Message of size %lu byte is too large for DTLS connection"
msgid_plural "Message of size %lu bytes is too large for DTLS connection"
msgstr[0] "A(z) %lu bájt méretű üzenet túl nagy a DTLS kapcsolathoz"
msgstr[1] "A(z) %lu bájt méretű üzenet túl nagy a DTLS kapcsolathoz"

#: tls/gnutls/gtlsconnection-gnutls.c:1264
#, c-format
msgid "(maximum is %u byte)"
msgid_plural "(maximum is %u bytes)"
msgstr[0] "(legfeljebb %u bájt)"
msgstr[1] "(legfeljebb %u bájt)"

#: tls/gnutls/gtlsconnection-gnutls.c:1311
#, c-format
msgid "Error performing TLS close: %s"
msgstr "Hiba a TLS-lezárás végrehajtásakor: %s"

#: tls/gnutls/gtlsdatabase-gnutls.c:575
msgid ""
"Failed to load system trust store: GnuTLS was not configured with a system "
"trust"
msgstr ""
"Nem sikerült betölteni a rendszer megbízhatósági tárát: a GnuTLS nem "
"rendszer megbízhatósággal lett beállítva"

#: tls/gnutls/gtlsdatabase-gnutls.c:580 tls/openssl/gtlsdatabase-openssl.c:187
#, c-format
msgid "Failed to load system trust store: %s"
msgstr "Nem sikerült betölteni a rendszer megbízhatósági tárát: %s"

#: tls/gnutls/gtlsfiledatabase-gnutls.c:153
#: tls/openssl/gtlsfiledatabase-openssl.c:454
#, c-format
msgid "Failed to populate trust list from %s: %s"
msgstr "Nem sikerült feltölteni a megbízhatósági listát innen: %s: %s"

#: tls/gnutls/gtlsserverconnection-gnutls.c:124
#: tls/openssl/gtlsserverconnection-openssl.c:170
#: tls/openssl/gtlsserverconnection-openssl.c:226
msgid "Certificate has no private key"
msgstr "A tanúsítványnak nincs személyes kulcsa"

#: tls/openssl/gtlsclientconnection-openssl.c:308
#: tls/openssl/gtlsserverconnection-openssl.c:344
#, c-format
#| msgid "Could not create TLS context: %s"
msgid "Could not set TLS cipher list: %s"
msgstr "Nem sikerült beállítani a TLS titkosítási listát: %s"

#: tls/openssl/gtlsclientconnection-openssl.c:324
#: tls/openssl/gtlsserverconnection-openssl.c:360
#, c-format
msgid "Could not set MAX protocol to %ld: %s"
msgstr "A MAX protokoll nem állítható erre: %ld: %s"

#: tls/openssl/gtlsclientconnection-openssl.c:377
#: tls/openssl/gtlsserverconnection-openssl.c:413
#, c-format
msgid "Could not create TLS context: %s"
msgstr "Nem sikerült létrehozni TLS-környezetet: %s"

#: tls/openssl/gtlsconnection-openssl.c:197
#, c-format
msgid "Unacceptable TLS certificate authority"
msgstr "Elfogadhatatlan TLS hitelesítésszolgáltató"

#: tls/openssl/gtlsconnection-openssl.c:205
msgid "Digest too big for RSA key"
msgstr "A kivonat túl nagy az RSA kulcshoz"

#: tls/openssl/gtlsconnection-openssl.c:213
msgid "Secure renegotiation is disabled"
msgstr "A biztonságos újraegyeztetés le van tiltva"

#: tls/openssl/gtlsconnection-openssl.c:234
#, c-format
msgid "%s: The connection is broken"
msgstr "%s: A kapcsolat megszakadt"

#: tls/openssl/gtlsconnection-openssl.c:489
#, c-format
msgid "Channel binding data tls-unique is not available"
msgstr "A tls-unique csatornakötési adatok nem érhetők el"

#: tls/openssl/gtlsconnection-openssl.c:512
#, c-format
msgid "X.509 Certificate is not available on the connection"
msgstr "Az X.509 tanúsítvány nem érhető el a kapcsolaton"

#: tls/openssl/gtlsconnection-openssl.c:558
#, c-format
msgid "Failed to generate X.509 certificate digest"
msgstr "Az X.509 tanúsítványkivonat előállítása sikertelen"

#: tls/openssl/gtlsconnection-openssl.c:589
#, c-format
msgid "TLS Connection does not support TLS-Exporter feature"
msgstr "A TLS kapcsolat nem támogatja a TLS exportáló funkciót"

#: tls/openssl/gtlsconnection-openssl.c:592
#, c-format
msgid "Unexpected error while exporting keying data"
msgstr "Váratlan hiba a kulcsadatok exportálásakor"

#: tls/openssl/gtlsconnection-openssl.c:833
msgid "Error performing TLS close"
msgstr "Hiba a TLS lezárás végrehajtásakor"

#: tls/openssl/gtlsdatabase-openssl.c:227
msgid "Could not create CA store"
msgstr "Nem sikerült létrehozni a CA-tárat"

#: tls/openssl/gtlsserverconnection-openssl.c:177
#: tls/openssl/gtlsserverconnection-openssl.c:245
#, c-format
msgid "There is a problem with the certificate private key: %s"
msgstr "Probléma van a tanúsítvány személyes kulcsával: %s"

#: tls/openssl/gtlsserverconnection-openssl.c:186
#: tls/openssl/gtlsserverconnection-openssl.c:237
#, c-format
msgid "There is a problem with the certificate: %s"
msgstr "Probléma van a tanúsítvánnyal: %s"

#~ msgid "Peer requested illegal TLS rehandshake"
#~ msgstr "A partner illegális ismételt TLS-kézfogást kért"

#~ msgid "Failed to load file path: %s"
#~ msgstr "Nem sikerült betölteni a fájlútvonalat: %s"
